utilities/EluvioLiveCli.js

```diff
@@ -2056,6 +2163,74 @@ yargs(hideBin(process.argv))
     }
   )
 
+  .command(
+    "tenant_show_new <tenant> [options]",
+    "Show info on this tenant",
+    (yargs) => {
+      yargs
+        .positional("tenant", {
+          describe: "Tenant ID",
+          type: "string",
+        });
+    },
+    (argv) => {
+      CmdTenantShowNew({ argv });
+    }
+  )
+
+  .command(
+    "tenant_fix <tenant> [options]",
+    "Fix tenant",
+    (yargs) => {
+      yargs
+        .positional("tenant", {
+          describe: "Tenant ID",
+          type: "string",
+        });
+    },
+    (argv) => {
+      CmdTenantFix({ argv });
+    }
+  )
+
+  .command(
+    "tenant_add_content_admin <tenant> <content_admin_address>",
```

Comment:
I am thinking it might be good to change every instance of `(content/tenant) admins address` that refers to a group to `(content/tenant) admin group address` - might be more clear this way, because it is the groups that are located in these addresses.

Reply:
Also, I noticed there are other times where the code doesn't respect plurality, for example: [link](https://github.com/eluv-io/elv-live-js/blob/b1d11893bcd11af1b8599e6860b64e0b8dea7aea/src/EluvioLive.js#L77-L86). I think it's worth having a clean-up ticket, which can be assigned to me, or another PR to fix this.